frontend/src/index.tsx

```diff
@@ -25,7 +26,11 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
             children: [
               {
                 path: AppRoute.ROOT,
-                element: <PageLayout style="black">Root</PageLayout>,
+                element: (
+                  <ProtectedRoute>
+                    <PageLayout>Root</PageLayout>
```

Comment:
Guess `style` prop should not be removed
```suggestion
                    <PageLayout style="black">Root</PageLayout>
```

Reply:
**was not resolved**